recipe/meta.yaml

```diff
@@ -35,21 +39,54 @@ requirements:
     # need a CDT for libudev for cos6, proper package available for cos7 (below)
     - {{ cdt('libudev-devel') }}  # [linux and cdt_name=='cos6']
   host:
+    - python
     - libudev  # [linux and cdt_name!='cos6']
     - libusb
   run:
     - __cuda  # [cuda_compiler_version != "None"]
 
-test:
-  commands:
-    - test -f ${PREFIX}/include/librealsense2/rs.hpp  # [not win]
-    - test -f ${PREFIX}/lib/librealsense2.so  # [linux]
-    - test -f ${PREFIX}/lib/librealsense2.dylib  # [osx]
-    - test -f ${PREFIX}/lib/cmake/realsense2/realsense2Config.cmake  # [not win]
-    - if exist %PREFIX%\\Library\\include\\librealsense2\\rs.hpp (exit 0) else (exit 1)  # [win]
-    - if exist $PREFIX$\\Library\\lib\\realsense2.lib (exit 0) else (exit 1)  # [win]
-    - if exist $PREFIX$\\Library\\bin\\realsense2.dll (exit 0) else (exit 1)  # [win]
-    - if exist %PREFIX%\\Library\\lib\cmake\\realsense2\\realsense2Config.cmake (exit 0) else (exit 1)  # [win]
+
+outputs:
+  - name: librealsense
+    build:
+      string: py{{ PY_VER_MAJOR }}{{ PY_VER_MINOR }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
+    requirements:
+      host:
+        # Explicitely add a python requirement so that the hash changes between
+        # pyrealsense for pypy36 and python3.6
+        # similarly for pypy37 and python3.7
+        - python
+        - libudev  # [linux and cdt_name!='cos6']
+        - libusb
+    test:
+      commands:
+        - test -f ${PREFIX}/include/librealsense2/rs.hpp  # [not win]
+        - test -f ${PREFIX}/lib/librealsense2.so  # [linux]
+        - test -f ${PREFIX}/lib/librealsense2.dylib  # [osx]
+        - test -f ${PREFIX}/lib/cmake/realsense2/realsense2Config.cmake  # [not win]
+        - if exist %PREFIX%\\Library\\include\\librealsense2\\rs.hpp (exit 0) else (exit 1)  # [win]
+        - if exist $PREFIX$\\Library\\lib\\realsense2.lib (exit 0) else (exit 1)  # [win]
+        - if exist $PREFIX$\\Library\\bin\\realsense2.dll (exit 0) else (exit 1)  # [win]
+        - if exist %PREFIX%\\Library\\lib\cmake\\realsense2\\realsense2Config.cmake (exit 0) else (exit 1)  # [win]
+
+  - name: pyrealsense2
+    build:
+      string: py{{ PY_VER_MAJOR }}{{ PY_VER_MINOR }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
+      run_exports:
+        - {{ pin_subpackage('pyrealsense2') }}
```

Comment:
```suggestion
```

I know that opencv recipe also has a run_exports for the Python package, but that does not make a lot of sense, so let's remove it, thanks!